### PR TITLE
Polishing RabbitAutoConfigurationTests to use kebab-case keys

### PR DESCRIPTION
**Pull request by quaff**

### CODE REVIEW

File: spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/amqp/RabbitAutoConfigurationTests.java [hunk 1]
```diff
@@ -894,18 +894,18 @@ void enableSslWithBundle() {
 	void enableSslWithKeystoreTypeAndTrustStoreTypeShouldWork() {
 		this.contextRunner.withUserConfiguration(TestConfiguration.class)
 			.withPropertyValues("spring.rabbitmq.ssl.enabled:true",
-					"spring.rabbitmq.ssl.keyStore=/org/springframework/boot/autoconfigure/amqp/test.jks",
-					"spring.rabbitmq.ssl.keyStoreType=jks", "spring.rabbitmq.ssl.keyStorePassword=secret",
-					"spring.rabbitmq.ssl.trustStore=/org/springframework/boot/autoconfigure/amqp/test.jks",
-					"spring.rabbitmq.ssl.trustStoreType=jks", "spring.rabbitmq.ssl.trustStorePassword=secret")
+					"spring.rabbitmq.ssl.key-store=/org/springframework/boot/autoconfigure/amqp/test.jks",
+					"spring.rabbitmq.ssl.key-store-type=jks", "spring.rabbitmq.ssl.keyStorePassword=secret",
```
Comment (Bursul) [inline spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/amqp/RabbitAutoConfigurationTests.java:898]: `spring.rabbitmq.ssl.keyStorePassword` should also be modified

File: spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/amqp/RabbitAutoConfigurationTests.java [hunk 2]
```diff
@@ -894,18 +894,18 @@ void enableSslWithBundle() {
...
+					"spring.rabbitmq.ssl.trust-store=/org/springframework/boot/autoconfigure/amqp/test.jks",
+					"spring.rabbitmq.ssl.trust-store-type=jks", "spring.rabbitmq.ssl.trustStorePassword=secret")
```
Comment (Bursul) [inline spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/amqp/RabbitAutoConfigurationTests.java:900]: `spring.rabbitmq.ssl.trustStorePassword` should also be modified

File: spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/amqp/RabbitAutoConfigurationTests.java [hunk 3]
```diff
@@ -955,9 +955,9 @@ void enableSslWithInvalidKeyStoreAlgorithmShouldFail() {
 	void enableSslWithInvalidTrustStoreAlgorithmShouldFail() {
 		this.contextRunner.withUserConfiguration(TestConfiguration.class)
 			.withPropertyValues("spring.rabbitmq.ssl.enabled:true",
-					"spring.rabbitmq.ssl.trustStore=/org/springframework/boot/autoconfigure/amqp/test.jks",
-					"spring.rabbitmq.ssl.trustStoreType=jks", "spring.rabbitmq.ssl.trustStorePassword=secret",
-					"spring.rabbitmq.ssl.trustStoreAlgorithm=test-invalid-algo")
+					"spring.rabbitmq.ssl.trust-store=/org/springframework/boot/autoconfigure/amqp/test.jks",
+					"spring.rabbitmq.ssl.trust-store-type=jks", "spring.rabbitmq.ssl.trustStorePassword=secret",
```
Comment (Bursul) [inline spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/amqp/RabbitAutoConfigurationTests.java:959]: `spring.rabbitmq.ssl.trustStorePassword` should also be modified

File: spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/amqp/RabbitAutoConfigurationTests.java [hunk 4]
```diff
@@ -928,22 +928,22 @@ void enableSslWithValidateServerCertificateDefault(CapturedOutput output) {
...
 
 	@Test
 	void enableSslWithInvalidKeyStoreAlgorithmShouldFail() {
 		this.contextRunner.withUserConfiguration(TestConfiguration.class)
 			.withPropertyValues("spring.rabbitmq.ssl.enabled:true",
-					"spring.rabbitmq.ssl.keyStore=/org/springframework/boot/autoconfigure/amqp/test.jks",
-					"spring.rabbitmq.ssl.keyStoreType=jks", "spring.rabbitmq.ssl.keyStorePassword=secret",
-					"spring.rabbitmq.ssl.keyStoreAlgorithm=test-invalid-algo")
+					"spring.rabbitmq.ssl.key-store=/org/springframework/boot/autoconfigure/amqp/test.jks",
+					"spring.rabbitmq.ssl.key-store-type=jks", "spring.rabbitmq.ssl.keyStorePassword=secret",
```
Comment (Bursul) [inline spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/amqp/RabbitAutoConfigurationTests.java:945]: `spring.rabbitmq.ssl.keyStorePassword `should also be modified

File: spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/amqp/RabbitAutoConfigurationTests.java [hunk 5]
```diff
@@ -928,22 +928,22 @@ void enableSslWithValidateServerCertificateDefault(CapturedOutput output) {
...
-					"spring.rabbitmq.ssl.keyStoreType=jks", "spring.rabbitmq.ssl.keyStorePassword=secret",
-					"spring.rabbitmq.ssl.keyStoreAlgorithm=PKIX",
-					"spring.rabbitmq.ssl.trustStore=/org/springframework/boot/autoconfigure/amqp/test.jks",
-					"spring.rabbitmq.ssl.trustStoreType=jks", "spring.rabbitmq.ssl.trustStorePassword=secret",
-					"spring.rabbitmq.ssl.trustStoreAlgorithm=PKIX")
+					"spring.rabbitmq.ssl.key-store=/org/springframework/boot/autoconfigure/amqp/test.jks",
+					"spring.rabbitmq.ssl.key-store-type=jks", "spring.rabbitmq.ssl.keyStorePassword=secret",
+					"spring.rabbitmq.ssl.key-store-algorithm=PKIX",
+					"spring.rabbitmq.ssl.trust-store=/org/springframework/boot/autoconfigure/amqp/test.jks",
+					"spring.rabbitmq.ssl.trust-store-type=jks", "spring.rabbitmq.ssl.trustStorePassword=secret",
```
Comment (Bursul) [inline spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/amqp/RabbitAutoConfigurationTests.java:935]: `spring.rabbitmq.ssl.trustStorePassword` should also be modified

File: spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/amqp/RabbitAutoConfigurationTests.java [hunk 6]
```diff
@@ -928,22 +928,22 @@ void enableSslWithValidateServerCertificateDefault(CapturedOutput output) {
...
 		this.contextRunner.withUserConfiguration(TestConfiguration.class)
 			.withPropertyValues("spring.rabbitmq.ssl.enabled:true",
-					"spring.rabbitmq.ssl.keyStore=/org/springframework/boot/autoconfigure/amqp/test.jks",
-					"spring.rabbitmq.ssl.keyStoreType=jks", "spring.rabbitmq.ssl.keyStorePassword=secret",
-					"spring.rabbitmq.ssl.keyStoreAlgorithm=PKIX",
-					"spring.rabbitmq.ssl.trustStore=/org/springframework/boot/autoconfigure/amqp/test.jks",
-					"spring.rabbitmq.ssl.trustStoreType=jks", "spring.rabbitmq.ssl.trustStorePassword=secret",
-					"spring.rabbitmq.ssl.trustStoreAlgorithm=PKIX")
+					"spring.rabbitmq.ssl.key-store=/org/springframework/boot/autoconfigure/amqp/test.jks",
+					"spring.rabbitmq.ssl.key-store-type=jks", "spring.rabbitmq.ssl.keyStorePassword=secret",
```
Comment (Bursul) [inline spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/amqp/RabbitAutoConfigurationTests.java:932]: `spring.rabbitmq.ssl.keyStorePassword `should also be modified

**Comment by philwebb** [comment]
Thanks @quaff and @Bursul

**Pull request closed without merging**